### DISCUSSION TITLE
Qodana GitHub action pull a wrong docker image

### DISCUSSION THREAD

**Discussion by qhy040404** [JetBrains/Qodana#113] [Q&A] [answered]
https://github.com/qhy040404/Library-One-Tap-Android/runs/7725749898?check_suite_focus=true

This is an Android project, but the action pulls qodana-jvm-community.
qodana-jvm-android should be used.
And qodana only checks my xml files but not kt or java

**Comment by brichbash** [comment, marked as answer]
Hello,
Please use `args` option with flags from [Qodana CLI](https://github.com/jetbrains/qodana-cli#options-1). To use android linter, add to your configuration:
```
 uses: JetBrains/qodana-action@v2022.2.1
        with:
          args: -l,jetbrains/qodana-jvm-android
```

**Reply by qhy040404** [reply]
Yep! That's the solution.
